src/App/App.tsx

```diff
@@ -44,6 +44,7 @@ export default function App() {
   return (
     <ModalContext>
       <SentryRoutes>
+        <Route path={`${appRoutes.donate}/:id`} element={<Donate />} />
```

Comment:
Move `/Donate` to it's own layout instead of it just covering `header | main | footer` this prevents z-index clashes with the header dropdowns

Reply:
Great idea!